integration-tests/nitrite/src/main/java/org/apache/camel/quarkus/component/nitrite/it/NitriteResource.java

```diff
@@ -84,11 +92,12 @@ public Object postRepositoryClass(Employee object) {
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
-    public Object postRepositoryClassOperation(Operation operation) {
+    public Object postRepositoryClassOperation(Operation operation, @QueryParam("mappable") boolean mappable) {
+        String className = mappable ? EmployeeMappable.class.getName() : EmployeeSerializable.class.getName();
         LOG.debugf("Sending to nitrite: {%s}", operation);
         return producerTemplate.toF("nitrite://%s?repositoryClass=%s",
-                dbFile, Employee.class.getName())
-                .withBody(operation.getEmployee())
+                dbFile, className)
+                .withBody(mappable ? operation.getEmployeeMappable() : operation.getEmployeeSerializable())
```

Comment:
I do not see `getCollection()` and `postCollection` being used anywhere. Maybe you wanted to add more tests?

Reply:
This is only difference in method names and mapped url. Both methods are used. Names are fixed, thanks.

Reply:
Thanks, I can see the test now. 

I was CTRL-F-ing for `/collection` in browser and for some reason it was not finding anything.